CHANGELOG.md

```diff
@@ -234,6 +234,16 @@ Previously the default value for `.editorconfig` property `max_line_length` was
 * Add new experimental rule `no-blank-line-in-list` for `ktlint_official` code style. This rule disallows blank lines to be used in super type lists, type argument lists, type constraint lists, type parameter lists, value argument lists, and value parameter lists. This rule can also be run for other code styles but then its needs to be explicitly enabled. ([#1224](https://github.com/pinterest/ktlint/issues/1224))
 * Add new experimental rule `multiline-expression-wrapping` for `ktlint_official` code style. This forces a multiline expression as value in an assignment to start on a separate line. This rule can also be run for other code styles but then its needs to be explicitly enabled. ([#1217](https://github.com/pinterest/ktlint/issues/1217))
 * Add new experimental rule `string-template-indent` for `ktlint_official` code style. This forces multiline string templates which are post-fixed with `.trimIndent()` to be formatted consistently. The opening and closing `"""` are placed on separate lines and the indentation of the content of the template is aligned with the `"""`. This rule can also be run for other code styles but then its needs to be explicitly enabled. ([#925](https://github.com/pinterest/ktlint/issues/925))
+* Support Bill of Materials (BOM), now you can integrate Ktlint in your `build.gradle` like:
+```kotlin
+dependencies {
+    implementation(platform("com.pinterest:ktlint-bom:0.49.0"))
+    implementation("com.pinterest:ktlint")
+    implementation("com.pinterest:ktlint-core")
+    implementation("com.pinterest:ktlint-ruleset-standard")
+    ...
+}
+```
```

Comment:
This should also be added to the Ktlint documentation. After I merged #1881 this should be added to https://github.com/pinterest/ktlint/blob/2c6040e484d4197679084b1002511a0923224681/docs/api/custom-integration.md

Reply:
Agreed.

Reply:
Might be better to update https://github.com/pinterest/ktlint/blob/master/docs/install/integrations.md#custom-gradle-integration ?

Reply:
Good catch. But I do think that we need it on both pages. I will take care of the API page. Btw, I would like that `implementation("com.pinterest:ktlint")` is removed from the example above. It can result in adding dependencies like logback to your project and result in conflict with logging.

Reply:
5fb736e28c721bae9be0ce9a356744434bb4b942

Reply:
I think only Gradle plugins need `implementation` ktlint deps in their projects, like [Kotlinter did](https://github.com/jeremymailen/kotlinter-gradle/blob/5c890e84bcc1b19bbaf92f5b50a9505c4b3e3807/build.gradle.kts#L56-L67), not sure how to update integration docs.

Reply:
Neither do I at this moment. So we keep the docs as they are right now. We can change them later when we know how to do it.